### ISSUE TITLE
Follow Unfollow Button next to Mutuals/Following/Follows you label

### ISSUE THREAD

**Issue by patak-cat** [elk-zone/elk#103]
I asked about our current UI here: https://mas.to/@patak/109399921162411092
I think we can keep "Mutuals" meaning "Following + Follows you", but the label transforming into an action button is not accessible and it doesn't work well on mobile.

I'm thinking we should add an Icon button for Follow/unfollow next to the label. Going to play with this a bit. Creating an issue in case someone has other ideas.
